tests/unit/__init__.py

```diff
@@ -1 +1,10 @@
- 
+# -*- coding: utf-8 -*-
```

Comment:
Can you move that function to the testlib module too? so we don't need the extra init file?